websites/api-docs-smoke-test/src/@commercetools-docs/gatsby-theme-docs/overrides/page-header-side.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import SpacingsStack from '@commercetools-uikit/spacings-stack';
+import { UserResearchBanner } from '@commercetools-docs/gatsby-theme-docs';
+import GitHubIcon from '../../../icons/github.svg';
+
+// A React component to be rendered in the top-right corner of a content page
+// eslint-disable-next-line react/display-name
+export default () => {
+  return (
+    <SpacingsStack size="m">
+      <div>
+        <GitHubIcon />
+      </div>
+      <UserResearchBanner />
```

Comment:
Sorry for the back and forth.

The banner should be included in the docs theme by default, and should be rendered as a separate overrides here:

https://github.com/commercetools/commercetools-docs-kit/blob/b4063e6518eb4398b6a8f8e75ea7fb6739795d01/packages/gatsby-theme-docs/src/layouts/content.js#L56-L58

```js
<LayoutPageHeaderSide>
  <PlaceholderPageHeaderSide />
  <PlaceholderPageHeaderSideBannerArea />
</LayoutPageHeaderSide>
```

Reply:
🙈